<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="foto_grande">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>130</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1392</width>
           <height>146</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="ClickeableLabel" name="poke_1">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/001.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ClickeableLabel" name="poke_2">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/004.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ClickeableLabel" name="poke_3">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/007.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ClickeableLabel" name="poke_4">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/010.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ClickeableLabel" name="poke_5">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/025.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ClickeableLabel" name="poke_6">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/027.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ClickeableLabel" name="poke_7">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/035.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ClickeableLabel" name="poke_8">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/029.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ClickeableLabel" name="poke_9">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/037.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ClickeableLabel" name="poke_10">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/050.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ClickeableLabel" name="poke_11">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/054.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ClickeableLabel" name="poke_12">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/058.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ClickeableLabel" name="poke_13">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../Downloads/066.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="actionCargar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionGuardar">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionCargar">
   <property name="text">
    <string>Print Image</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickeableLabel</class>
   <extends>QLabel</extends>
   <header>etiqueta_clicable</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
